homeassistant/components/madvr/strings.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "title": "Setup MadVR Envy",
+        "description": "Your device needs to be turned on so we can test the connection. ",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]"
+        }
```

Comment:
Also add a `data_description` part that explains a bit of context, as not everyone knows what a host is. Check other integrations for an example